src/core/Akka.MultiNodeTestRunner/README.md

```diff
@@ -8,13 +8,30 @@ This README explains how to run the `Akka.MultiNodeTestRunner` to execute any `M
 
-Right now the only options for running the `MultiNodeTestRunner` is to build from the Akka.NET source and manually copy the binaries out of `src\core\Akka.MultiNodeTestRunner\bin\[Debug|Release]`:
+Right now the only options for running the `MultiNodeTestRunner` is to build from the Akka.NET source and manually copy the binaries out of `src\core\Akka.MultiNodeTestRunner\bin\[Debug|Release]\[Platform name]`, where `[Platform name]` can be either `net5.0`, `net471`, or `netcoreapp3.1`:
 
 ![MultiNodeTestRunner binaries](../../../documentation/wiki/images/multinode-teskit/multi-node-testrunner-binaries.png)
 
-The `Akka.MultiNodeTestRunner` process requires only one argument - the full path or name of the assembly containing `MultiNodeSpec` tests.
-
-    C:> Akka.MultiNodeTestRunner.exe [assembly name]
+## Commandline Arguments
+
+```
+C:> Akka.MultiNodeTestRunner.exe [assembly name] [-Dmultinode.output-directory={dir-path}] [-Dmultinode.failed-specs-directory={folder-name}] [-Dmultinode.loglevel={DEBUG|INFO|WARNING|ERROR}] [-Dmultinode.listen-address={ip-address}] [-Dmultinode.listen-port={ip-port}] [-Dmultinode.reporter={trx|teamcity|console}] [-Dmultinode.clear-output={0|1}] [-Dmultinode.spec={spec-filter}] [-Dmultinode.include={include-filter}] [-Dmultinode.exclude={exclude-filter}] 
+```
+
+- __assembly name__ : The full path or name of the assembly containing `MultiNodeSpec` tests.
+- __-Dmultinode.output-directory__ : Folder where the test report will be exported. Default value is the current working directory.
+- __-Dmultinode.failed-specs-directory__ : Folder name inside the output directory where failed test log will be exported, if a test should fail. Default value is `FAILED_SPECS_LOG`.
+- __-Dmultinode.loglevel__ : Sets the minimum reported log level used within the test. Valid values are `DEBUG`, `INFO`, `WARNING`, and `ERROR`. Default value is `WARNING`.
+- __-Dmultinode.listen-address__ : The TCP/IP address the multi node test runner should listen for test node reports/logs. Default value is `127.0.0.1`.
+- __-Dmultinode.listen-port__ : The TCP/IP port the multi node test runner should listen for test node reports/logs. Default value is `6577`.
+- __-Dmultinode.reporter__ : The report type this runner should export in. Valid values are `trx`, `teamcity`, and `console`. Note that report files are exported to the current directory for `trx`. Default value is `console`.
+- __-Dmultinode.clear-output__ : This flag will clear the output folder before any test is run when it is set to 1. Default value is 0.
+- __-Dmultinode.spec__ : Apply a filter to the test class names within the dll. Any fully qualified test class name that contains this string will run. Default value is (all).
+- __-Dmultinode.include__ : a "`,`" (comma) separted list of wildcard pattern to be matched and included in the tests. Default value is `*` (all). The filter is applied on the complete name of the test `Namespace.Class+MethodName`
```

Comment:
Are you sure it's not just the `MethodName`?

Reply:
Yes, I've made sure of that.